pandas/core/internals.py

```diff
@@ -2603,6 +2586,12 @@ class DatetimeTZBlock(NonConsolidatableMixIn, DatetimeBlock):
     _concatenator = staticmethod(_concat._concat_datetime)
     is_datetimetz = True
 
+    get_values = DatetimeBlock.get_values  # override NonConsolidatableMixin
```

Comment:
I'd rather you write an actual function here (that can call DatetimeBlock.get_values), but that has a doc-string and such